src/effects/backends/effectmanifest.h

```diff
@@ -142,7 +154,7 @@ class EffectManifest final {
         m_metaknobDefault = metaknobDefault;
     }
 
-    QString backendName() {
+    QString backendName() const {
```

Comment:
should these be only used for display or debugging, or are they also useful internally? Some doc would help